bakerydemo/base/fixtures/bakerydemo.json

```diff
@@ -1552,6 +1671,96 @@
       "user_permissions": []
     }
   },
+  {
+    "model": "auth.user",
+    "pk": 4,
+    "fields": {
+      "password": "pbkdf2_sha256$600000$vDpqnaE6Nb6yc4r0v9Z6MK$xHyp2mJ1UZOhNnmjgOF8Id8W8gIbif8xyCS8Vj0NICU=",
+      "last_login": "2019-04-23T19:58:19.460Z",
+      "is_superuser": false,
+      "username": "editor",
+      "first_name": "Editor",
+      "last_name": "User",
+      "email": "editor@example.com",
+      "is_staff": false,
+      "is_active": true,
+      "date_joined": "2019-02-17T08:01:33.700Z",
+      "groups": [["Editors"]],
+      "user_permissions": []
+    }
+  },
```

Comment:
**Suggestion:** we should document that these users exist and why.

Reply:
I had a commit in my fork that does a similar thing here: laymonage/bakerydemo@f42d7d1491941fa5313c8475ea7e0074b24442dc

In my branch, I used these users for benchmarking the SQL queries (different permissions need different queries). They're also useful for testing workflows.

Definitely would be nice to document these.

To add, in my case I used names that are distinct from the group name, e.g. "Eddy Thor" Instead of "Editor User" just in case we need the distinction when we want to write tests.

Reply:
@laymonage are you suggesting that you want to rename the users? The current names feel adequate to me.

Reply:
👍 I’d say the more realistic the better. Real-world sites don’t have users named "editor".